spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -1703,18 +1703,21 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
               optExprWithInfo(optExpr, expr, child)
           }
 
+        /// The expression for `log` functions is defined as  null on numbers less than or equal
+        /// to 0.  This matches Spark and Hive behavior, where non positive values eval to null
+        /// instead of NaN or -Infinity
```

Comment:
CI is failing with scalastyle error `Insert a space after the start of the comment` for these lines. Changing `///` to `//` should resolve it.

```suggestion
        // The expression for `log` functions is defined as  null on numbers less than or equal
        // to 0.  This matches Spark and Hive behavior, where non positive values eval to null
        // instead of NaN or -Infinity
```